config/config_test.go

```diff
@@ -26,6 +26,55 @@ var _ = Describe("Config", func() {
 	})
 
 	Describe("#Validate", func() {
+		It("Returns no error if run_tcp_availability is set to true and tcp_domain and tcp_port are not provided", func() {
```

Comment:
This title is incorrect, the tcp_domain and tcp_port _are_ provided here.

Reply:
You're right, should be *are* provided

Reply:
This still says `not provided` instead of `provided`. Can you please fix this.